app/dashboard/tip_views.py

```diff
@@ -60,6 +61,45 @@ def send_tip(request):
     return TemplateResponse(request, 'onepager/send1.html', params)
 
 
+def request_money(request):
+    """"""
+    if request.method == 'POST':
+        username = request.POST.get('username', '').strip('@')
+        token_name = request.POST.get('tokenName')
+        amount = request.POST.get('amount')
+        comments = request.POST.get('comments')
+        network = request.POST.get('network')
+        address = request.POST.get('address')
+        token_address = request.POST.get('tokenAddress')
```

Comment:
Where is this used?

Reply:
Sorry, i forgot delete this unused variable